cli/prerender.js

```diff
@@ -0,0 +1,33 @@
+// Load zone.js for the server.
+require('zone.js/dist/zone-node');
+const fs = require('fs');
+
```

Comment:
Can you please add `reflect-metadata` here?
`require('reflect-metadata');`

I got the following error:

> node_modules/@angular/core/bundles/core.umd.js:382
>             throw 'reflect-metadata shim is required when using class decorators';
>             ^
> reflect-metadata shim is required when using class decorators
> 
You can also read the [Universal upgrade guide](https://github.com/angular/universal/blob/1aa8d582c1dcfcfe0f46cad69ff2593b38a8138b/UPGRADE-GUIDE.md#serverts) which they import `reflect-metadata`

Reply:
That's only if you run it in a none production mode

Reply:
Yes, but still it should work fine for both development and production mode with fewer changes.

Reply:
@gdi2290 What do you think?

Reply:
Sounds fair, added it in